server.js

```diff
@@ -13,6 +14,13 @@ const parser = require('yargs')
   .command('flip <text...>', 'flip text upside down', () => {}, (argv) => {
     argv.respond(flip(argv.text.join(' ')))
   })
+  .command('cowsay <text...>', 'Say it like a cow?', () => {}, (argv)={
```

Comment:
You mean _equal functions_ aren't a thing in JS anymore!? What has happened to this language?